mediaportal/MediaPortal.Base/skin/Titan/DialogPinCode.xml

```diff
@@ -1,145 +1,134 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <window>
-    <id>9915</id>
-    <defaultcontrol>10</defaultcontrol>
-    <allowoverlay>yes</allowoverlay>
-    <controls>
-        <import>common.dialog.xml</import>
-        <control>
-            <id>1</id>
-            <description>Lock</description>
-            <type>image</type>
-            <posX>850</posX>
-            <posY>266</posY>
-            <width>48</width>
-            <height>48</height>
-            <texture>lock.png</texture>
```

Comment:
Why did we remove this picture?

Reply:
This was copied from MovingPictures and looks better.